lib/Index/Index.cpp

```diff
@@ -450,6 +477,16 @@ class IndexSwiftASTWalker : public SourceEntityWalker {
     return nullptr;
   }
 
+  Decl *getContainingDecl() const {
+    auto Containers = (SymbolRoleSet)SymbolRole::Definition | (SymbolRoleSet)SymbolRole::Declaration;
+    for (const auto &Entity: EntitiesStack) {
```

Comment:
When does a declaration (not a definition) contain something?  Is this for default argument expressions or something?

Reply:
Nope, this just doesn't make sense 🙂
I don't even think we use the Declaration role in swift...